platform-sdk/docs/proposals/healthMonitor/healthMonitorProposal.md

```diff
@@ -0,0 +1,127 @@
+# Health Monitor Proposal
+
+---
+
+## Summary
+
+A replacement for the current hard backpressure mechanism using a health monitor.
```

Comment:
This reads to me that backpressure should be completely replaced by something different (called a health monitor). Later in the propasal I learned that this is just a name for more dynamic backpressure. Based on that the summary is misleading from my point of view,